packages/frontend/src/features/reporting/components/AdjudicateNotification/AdjudicateNotification.tsx

```diff
@@ -75,25 +77,63 @@ function useActiveAdjudication() {
 
 export default function AdjudicationNotification() {
     const { data: activeAdjudication, refetch } = useActiveAdjudication()
-
     const [open, toggle] = useToggle(false)
+    const [confirmOpen, setConfirmOpen] = useState(false)
+    const [buttonVisible, setButtonVisible] = useState(true)
 
-    const onClose = () => {
-        refetch()
+    useEffect(() => {
+        if (activeAdjudication) {
+            setButtonVisible(true)
+        }
+    }, [activeAdjudication])
+
+    const closeAdjudicate = () => {
         toggle(false)
     }
 
+    const closeConfirm = () => {
+        setConfirmOpen(false)
+    }
+
+    const handleCloseClick = () => {
```

Comment:
Suggest to rename this function to `openConfirmation` for clarity.

Reply:
fixed